pyproject.toml

```diff
@@ -123,6 +125,7 @@ all = [
     "markdownify",
     "voyageai",
     "elevenlabs",
+    "torch",
```

Comment:
Can you explain a bit why we need this now?

Reply:
Without this, running the example provided in the docs, I get:

```
/Users/dylan/Documents/boom/.venv/lib/python3.12/site-packages/stringcase.py:247: SyntaxWarning: invalid escape sequence '\W'
  return re.sub("\W+", "", string)
None of PyTorch, TensorFlow >= 2.0, or Flax have been found. Models won't be available and only tokenizers, configuration and file/data utilities can be used.
[05/21/24 19:26:43] INFO     PromptTask ed3273d35309405e87eaacc7bb3360ee
                             Input: Hello Girafatron, what is your favorite animal?
WARNING:root:<RetryCallState 6048187696: attempt #1; slept for 0.0; last result: failed (RuntimeError At least one of TensorFlow 2.0 or PyTorch should be installed. To install TensorFlow 2.0, read the instructions at https://www.tensorflow.org/install/ To install PyTorch, read the instructions at https://pytorch.org/.)>
WARNING:root:<RetryCallState 6048187696: attempt #2; slept for 2.0; last result: failed (RuntimeError At least one of TensorFlow 2.0 or PyTorch should be installed. To install TensorFlow 2.0, read the instructions at https://www.tensorflow.org/install/ To install PyTorch, read the instructions at https://pytorch.org/.)>
WARNING:root:<RetryCallState 6048187696: attempt #3; slept for 4.0; last result: failed (RuntimeError At least one of TensorFlow 2.0 or PyTorch should be installed. To install TensorFlow 2.0, read the instructions at https://www.tensorflow.org/install/ To install PyTorch, read the instructions at https://pytorch.org/.)>
```

I was surprised to see this as well, and I can't explain why now.